### DISCUSSION TITLE
"Собеседования"

### DISCUSSION THREAD

**Discussion by dirayser** [HowProgrammingWorks/LiveQA#1] [Q&A] [answered]
"Если кто-то работает по специальности, то я проведу краткое собеседование и все зачту, я так и раньше делал, я ставлю оченки за знания, а не за выполнение формальностей." 

Когда ждать подобные собеседования?

**Comment by tshemsedinov** [comment, marked as answer]
На любом из семинаров, просто подключаетесь с возможностью говорить и расшаривать экран (заранее проверьте микрофон и права).
